public/index.html

```diff
@@ -35,14 +35,24 @@
     <title>mutualaid.world</title>
   </head>
   <body>
-    <div id="fb-root"></div>
-    <script
-      async
-      defer
-      crossorigin="anonymous"
-      src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v6.0&appId=2470733643238869&autoLogAppEvents=1"
-    ></script>
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <script src="/__/firebase/7.14.3/firebase-app.js"></script>
 
+    <!-- TODO: Add SDKs for Firebase products that you want to use
+     https://firebase.google.com/docs/web/setup#available-libraries -->
+    <script src="/__/firebase/7.14.3/firebase-analytics.js"></script>
+    <script src="/__/firebase/7.14.3/firebase-auth.js"></script>
+    <script src="/__/firebase/7.14.3/firebase-firestore.js"></script>
+    <script src="/__/firebase/7.14.3/firebase-storage.js"></script>
```

Comment:
Switching the app to use firebase's hosted packages instead of the ones provided by npm.
This creates a global `firebase` variable on the window.

The reason for this is that it allows us to completely remove the firebase config and initializeApp calls as they're now auto configured by the firebase host, which means we can easily switch between as many different prod or staging environments as we want without having to worry about setting up the config for that project.